### PR TITLE
Update answer wording

### PR DESCRIPTION
**Pull request by shuoh**
@deyang Please review

### CODE REVIEW

File: data/price.qa [hunk 1]
```diff
@@ -3,22 +3,22 @@
 [
     [
         "How does it charge?",
-        "Buddy AI is currently in private beta and pricing is dependent on multiple variables. To learn more about our beta program, drop me an e-mail"
+        "Buddy AI is currently in private beta and the pricing is dependent on a few factors including the volume of customer questions and the size of your company. To learn more about our beta program, drop me an e-mail at info@everbuddy.io"
```
Comment (deyang) [inline data/price.qa]: We should have different prices for different sizes of companies if they have the same volume? So we are charging based on volume of questions and number of human agents? I think just saying the size of the company is weird. Why larger/smaller company will pay more/less?

File: data/company.qa [hunk 2]
```diff
@@ -5,50 +5,50 @@
 [
     [
         "What is Buddy AI?",
-        "Buddy AI is a startup building artificial intelligence based solutions for modernized customer communication. It provides an AI based chatbot that enables 24/7 customer support via a machine+human hybrid approach."
+        "Buddy AI is a startup company building AI-based solutions for modernized customer communication. It provides convenient and easy-to-use tools for businesses of all sizes to create AI chatbots that can respond to customer questions 24/7."
```
Comment (deyang) [inline data/company.qa]: I think what we provide is more than tools to create chatbots. We are not justing giving tools so the business can build their bot. We are providing chatbot as a service. The business does not need to build on their own. The word "tools" gives me an impression that there need to be a developer from the business side, who will get hands dirty by using our apis/consoles.

File: data/product.qa [hunk 3]
```diff
@@ -3,18 +3,18 @@
 [
     [
         "What are the technologies used here?",
-        "I'm using Natural Language Processing to understand your questions and machine learning to find the best answer"
+        "I don't know exactly how my creators built me, but I remember hearing a lot about NLP (Natural Language Processing) and machine learning."
     ],
     [
         "What are the features?",
-        "Buddy AI takes any ubiquitous data (forum, customer emails, dev docs, FAQs, internal knowledge base, etc), and then I will learn from the data and provide fully-automated or suggested answers to customer questions."
+        "Buddy AI bot can take any ubiquitous data as input for training (forum posts, customer emails, dev docs, FAQs, internal knowledge base, etc). Once trained, the bot can provide fully-automated or suggested answers to customer questions."
```
Comment (deyang) [inline data/product.qa]: forum discussions

**Comment by deyang** [comment]
Overall is nice. I want to think about more second degree questions. Maybe after we merge this PR.

File: data/price.qa (hunk 1 above)
Comment (shuoh) [inline data/price.qa]: I'm fine removing it. But I'm curious what are the other variables when you mentioned "multiple variables"?

File: data/company.qa (hunk 2 above)
Comment (shuoh) [inline data/company.qa]: Agreed.

File: data/product.qa (hunk 3 above)
Comment (shuoh) [inline data/product.qa]: okay

**Comment by shuoh** [comment]
> I want to think about more second degree questions. Maybe after we merge this PR.

Yes, that's the plan.

**Pull request closed without merging**